components/fontloader/fontloader.cpp

```diff
@@ -340,51 +340,48 @@ namespace Gui
             code->addAttribute("size", MyGUI::IntSize(static_cast<int>(data[i].width), static_cast<int>(data[i].height)));
 
             // More hacks! The french game uses several win1252 characters that are not included
-            // in the cp437 encoding of the font. Fall back to similar available characters.
-            if (mEncoding == ToUTF8::CP437)
+            // in the font. Fall back to similar available characters.
+            std::multimap<int, int> additional; // <cp437, unicode>
```

Comment:
IMO, CP437 comment is not needed now.
Also Morrowind supports the − U+2212 (MINUS SIGN) - some book mods use it.
Also it would be nice to add comments to all pairs, not just for 5 ones.

Reply:
We probably don't want to add fallbacks to the entire Unicode character table, though.